dhis-2/dhis-test-web-api/src/test/java/org/hisp/dhis/webapi/controller/tracker/export/event/EventsExportControllerTest.java

```diff
@@ -269,6 +287,68 @@ void getEventByIdWithDataValues() {
     assertHasMember(dataValue, "storedBy");
   }
 
+  @Test
+  void getEventsWithMultiTextDataValueUsingINOperator() {
```

Comment:
Can we have one test eacg that falls into each of the switch clauses above atleast.